tests/pass/warnings_and_errors.py

```diff
@@ -0,0 +1,18 @@
+import types
+from typing import Type
+
+import numpy as np
+
+warnings = np.warnings  # type: types.ModuleType
+ModuleDeprecationWarning = np.ModuleDeprecationWarning  # type: Type[DeprecationWarning]
```

Comment:
I would move these to `test/reveal/warnings_and_errors.py` and do something like

```
reveal_type(ModuleDeprecationWarning())  # E: numpy.DeprecationWarning
...
```